doozerlib/cli/release_gen_assembly.py

```diff
@@ -0,0 +1,296 @@
+import click
+import sys
+import json
+import yaml
+from typing import Dict, List, Set
+
+from doozerlib import util
+from doozerlib.cli import cli, pass_runtime
+from doozerlib import exectools
+from doozerlib.model import Model
+from doozerlib import brew
+from doozerlib.rpmcfg import RPMMetadata
+from doozerlib.image import ImageMetadata
+
+
+@cli.group("release:gen-assembly", short_help="Output assembly metadata based on inputs")
+@click.option('--name', metavar='ASSEMBLY_NAME', required=True, help='The name of the assembly (e.g. "4.9.99", "art1234") to scaffold')
+@click.pass_context
+def releases_gen_assembly(ctx, name):
+    ctx.ensure_object(dict)
+    ctx.obj['ASSEMBLY_NAME'] = name
+    pass
+
+
+@releases_gen_assembly.command('from-releases', short_help='Outputs assembly metadata based on a set of speified nightlies')
```

Comment:
The name 'from-releases' sounds like it is creating an assembly from multiple releases from releases.yml. What about renaming to `from-nightlies`?

Reply:
That works for me.